cvat-ui/src/components/create-model-page/create-model-content.tsx

```diff
@@ -18,8 +18,8 @@ import ConnectedFileManager, {
 } from 'containers/file-manager/file-manager';
 import { ModelFiles } from 'reducers/interfaces';
 
-import CreateModelForm, {
-    CreateModelForm as WrappedCreateModelForm,
+import WrappedCreateModelForm, {
```

Comment:
Default export in this file is not wrapped model form, is a wrapper for model form